authors/apps/profiles/apps.py

```diff
@@ -1,5 +0,0 @@
-from django.apps import AppConfig
```

Comment:
I thought you were to bring this lines back as per the former review

Reply:
And resolve conflicts

Reply:
I brought them back.

Reply:
I think when i rebased they disappered again

Reply:
Okay